c_tests/tests/compile_call_args.c

```diff
@@ -0,0 +1,34 @@
+// Compiler:
+// Run-time:
+
+// Check that basic trace compilation works.
+// FIXME An optimising compiler can remove all of the code between start/stop
+// tracing.
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <yk_testing.h>
+
+int f(int a, int b) {
```

Comment:
Again, probably worth marking as "don't inline".

Reply:
Fixed in 373a19d.